src/dialects/sqlite3/index.js

```diff
@@ -18,6 +19,9 @@ var SQLite3_DDL    = require('./schema/ddl')
 
 function Client_SQLite3(config) {
   Client.call(this, config)
+  if (!config.useNullAsDefault) {
```

Comment:
Hey, just looking at this (post merge, sorry). I think here it should be `if (_.isUndefined(config.useNullAsDefault))`. `false` is a valid option if user prefers to raise an error, but does not wish to see the warning.